feature-libs/product-configurator/common/assets/translations/en/configurator-common.ts

```diff
@@ -76,6 +76,12 @@ export const configurator = {
       cancelConfiguration: 'Cancel Configuration',
       cancelConfigurationMobile: 'Cancel',
     },
+    icon: {
+      groupComplete: 'Complete',
+      groupIncomplete: 'Incomplete',
+      groupConflict: 'Conflict',
+      subgroup: 'Sub-group',
```

Comment:
As the ">" icon means there is a sub-group, so the text of the tooltip should be something like "Has sub-groups" or similar

Reply:
changed to "has sub-groups"